Description: Update to Sphinx 1.5 searchindex.js
 Filename paths also need to be updated, otherwise search snippets won't be
 able to find the source file and "404" will be displayed instead. (The link
 to the html file still works.)
Author: Dmitry Shachnev <mitya57@debian.org>
Bug: https://trac.sagemath.org/ticket/23364
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/sage/src/sage_setup/docbuild/ext/multidocs.py
+++ b/sage/src/sage_setup/docbuild/ext/multidocs.py
@@ -148,7 +148,7 @@
             # merge the filenames
             filenames = app.builder.indexer._filenames
             for (res, filename) in six.iteritems(index._filenames):
-                filenames[fixpath(res)] = filename
+                filenames[fixpath(res)] = fixpath(filename)
             # TODO: merge indexer._objtypes, indexer._objnames as well
 
             # Setup source symbolic links
